lists.go

```diff
@@ -25,3 +30,19 @@ func (s *Session) GetUsers(l *List) (users []User, err error) {
 	err = s.do(rq).Into(&users)
 	return
 }
+
+// OptIn subscribes the given user to the given list
+// This consumes one API request.
+func (s *Session) OptIn(UserID int, ListID int) (err error) {
+	var body bytes.Buffer
+	var ok *bool
+	body.WriteString(fmt.Sprintf("userid=%d", UserID))
+	err = s.putf("/list/%d/optin", body, ListID).Into(&ok)
+	if err != nil {
```

Comment:
Might be able to roll this and the last line up into:

```go
if err = s.putf("/list/%d/optin", body, ListID).Into(&ok); err != nil 
    return
}
```

Second opinion on whether this is a good idea?